### PR TITLE
feat : add pgvector store example

### PR DESCRIPTION
**Pull request by rajadilipkolli**
This add pgvector embedding store example

### CODE REVIEW

**Comment by rajadilipkolli** [comment]
Hi @langchain4j , Any update on this PR?

File: postgres-example/src/main/java/PGVectorEmbeddingStoreExample.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import dev.langchain4j.store.embedding.pgvector.PgVectorEmbeddingStore;
+
+import java.util.List;
+
+public class PGVectorEmbeddingStoreExample {
+
+    public static void main(String[] args) {
+
+        PostgreSQLContainer<?> postgreSQLContainer = new PostgreSQLContainer<>(
+                DockerImageName.parse("ankane/pgvector:v0.5.1").asCompatibleSubstituteFor("postgres"));
```
Comment (eddumelendez) [inline postgres-example/src/main/java/PGVectorEmbeddingStoreExample.java:19]: pgvector/pgvector image can be used directly using Testcontainers 1.19.7

File: pom.xml [hunk 2]
```diff
@@ -24,6 +24,7 @@
         <module>opensearch-example</module>
         <module>other-examples</module>
         <module>pinecone-example</module>
+        <module>postgres-example</module>
```
Comment (langchain4j) [inline pom.xml:27]:
```suggestion
        <module>pgvector-example</module>
```

**Review by langchain4j — COMMENTED** [review]
@rajadilipkolli thanks a lot!

File: postgres-example/src/main/java/PGVectorEmbeddingStoreExample.java (hunk 1 above)
Comment (rajadilipkolli) [reply postgres-example/src/main/java/PGVectorEmbeddingStoreExample.java]: Thanks @eddumelendez , implemented the change

File: pom.xml (hunk 2 above)
Comment (rajadilipkolli) [reply pom.xml]: Hi @langchain4j , implemented both review comments.

**Review by langchain4j — APPROVED** [review]
@rajadilipkolli thank you!

**Comment by rajadilipkolli** [comment]
Hi @langchain4j , how to add above example to this [link ](https://docs.langchain4j.dev/tutorials/embedding-stores)under documentation

**Pull request merged**